src/utils.js

```diff
@@ -388,6 +388,25 @@ go.utils = {
             });
     },
 
+    send_welcome_sms: function(im, $) {
```

Comment:
because this is in utils let's make it more reusable and make it a `vumi_send_text` function that takes `to_addr` and `content` as params